src/scores/probability/crps_impl.py

```diff
@@ -947,6 +960,8 @@ def tw_crps_for_ensemble(
         weights: Weights for calculating a weighted mean of individual scores. Note that
             these weights are different to threshold weighting which is done by decision
             threshold.
+        decomposition: If True, returns the twCRPS with underforecast and overforecast
```

Comment:
see comment above about different decompositions that give different under and over forecast penalties. The same applies here for tw_crps_for_ensemble and twcrps for cdfs.